RNWCPP/ReactUWP/Views/ScrollViewManager.cpp

```diff
@@ -28,6 +30,289 @@ enum class ScrollViewCommands
   ScrollToEnd,
 };
 
+class ScrollViewShadowNode : public ShadowNodeBase
+{
+  using Super = ShadowNodeBase;
+public:
+  ScrollViewShadowNode();
+  void dispatchCommand(int64_t commandId, const folly::dynamic& commandArgs) override;
+  void createView() override;
+  void updateProperties(const folly::dynamic&& props) override;
+
+  // snaps to the snapToInterval based on the snapToAlignment (start, center, end) within the visable area as defined by the viewableSize
+  int snap(double original, winrt::Primitives::SnapPointsAlignment snapToAlignment, double snapToInterval, double viewableSize);
+
+protected:
```

Comment:
Why are these protected instead of private?

Reply:
I always use protected as default. I will move them to private